src/DIRAC/WorkloadManagementSystem/DB/PilotAgentsDB.py

```diff
@@ -639,26 +639,29 @@ def getGroupedPilotSummary(self, selectDict, columnList):
     Aborted pilots only from the last day. The selection is done entirely in SQL.
 
     :param dict selectDict: A dictionary to pass additional conditions to select statements, i.e.
-                            it allows to define start time for Done and Aborted Pilots.
+                            it allows to define start time for Done and Aborted Pilots. Unused.
```

Comment:
mmm?

Reply:
OK, I told the truth.. This parameter was present (and unused) in `PilotSummaryWeb` so I kept it. It can in fact serve more purposes that outlined in the doc above. I still need to change the signature of the above method to include `starItem` and `maxItems` which are necessary for the WebApp (only). I thought it would come together with a WebApp change.

Reply:
So, what do you think? To merge this PR now, or wait also that a WebApp change is coded?

Reply:
It's really only a fix of logging, and a slight modification of the query. I would merge it and start a new PR for the WebApp.